index.html

```diff
@@ -204,5 +204,66 @@ <h3>About Me</h3>
         <li class="my_skills">Terminal</li>
       </ul>
     </section>
+    <section class="motivation">
+      <p>
+        I'm always<br />
+        interested in hearing <br />
+        about new projects,<br />
+        so if you'd like to chat please get in <br />
+        touch.
+      </p>
+    </section>
+    <section class="form">
+      <form action="https://formspree.io/f/xeqnejbr" method="post">
+        <div>
+          <input
+            type="text"
+            id="full_name"
+            placeholder="Full name"
+            name="Full_name"
+            maxlength="30"
+            required
+          />
+        </div>
+        <div>
+          <input type="email" id="email" placeholder="Email" name="Email" />
+        </div>
+        <div>
+          <textarea
+            id="comment"
+            placeholder="Enter text here"
+            name="Comment"
+            maxlength="500"
+            required
+          ></textarea>
+        </div>
+        <div class="comment_button">
+          <button type="submit">Get in touch</button>
+        </div>
+      </form>
+      <div class="rectangle">
+        <hr class="footer_line" />
+      </div>
+    </section>
+    <section class="footer">
+      <ul class="socials_bottom">
+        <li>
+          <a href=""><i class="fab fa-github"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-linkedin-in"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-angellist"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fab fa-medium-m"></i></a>
+        </li>
+      </ul>
+      <hr class="bottom_footer_line" />
+    </section>
```

Comment:
- [ ] Please let's make this section match the template by making the lines centered properly.

Yours:
<img width="201" alt="Loom_YtyARxcZBU" src="https://user-images.githubusercontent.com/81468675/149136249-c7c79b6f-dedc-4e78-b214-4e5136cfe413.png">

Your template:
<img width="235" alt="Loom_WZlEJZgPTg" src="https://user-images.githubusercontent.com/81468675/149136499-f52d249a-628f-4a54-9152-afab21b447d8.png">